jb/itch5/compute_book_depth.hpp

```diff
@@ -0,0 +1,214 @@
+#ifndef jb_itch5_compute_book_depth_hpp
+#define jb_itch5_compute_book_depth_hpp
+
+#include <jb/itch5/order_book_depth.hpp>
+#include <jb/itch5/add_order_message.hpp>
+#include <jb/itch5/add_order_mpid_message.hpp>
+#include <jb/itch5/order_cancel_message.hpp>
+#include <jb/itch5/order_delete_message.hpp>
+#include <jb/itch5/order_executed_message.hpp>
+#include <jb/itch5/order_executed_price_message.hpp>
+#include <jb/itch5/order_replace_message.hpp>
+#include <jb/itch5/stock_directory_message.hpp>
+#include <jb/itch5/unknown_message.hpp>
+
+#include <boost/functional/hash.hpp>
+#include <chrono>
+#include <functional>
+#include <unordered_map>
+
+namespace jb {
+namespace itch5 {
+
+/* Ticket https://github.com/coryan/jaybeams/issues/20
```

Comment:
There are several of these comments that sound like "note to self", and should be deleted in the final version of the code.